pkg/nvpci/mock.go

```diff
@@ -84,6 +84,13 @@ func (m *MockNvpci) AddMockA100(address string, numaNode int, sriov *SriovInfo)
 		return err
 	}
 
+	vfioDev := filepath.Join(deviceDir, "vfio-dev")
+	vfioFD := filepath.Join(vfioDev, "vfio8")
+	err = os.MkdirAll(vfioFD, 0755)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: Is there a benefit in having two variables?
```suggestion
	vfioFD := filepath.Join(deviceDir, "vfio-dev", "vfio8)
	err = os.MkdirAll(vfioFD, 0755)
	if err != nil {
		return err
	}
```